example_client/stateful/README.md

```diff
@@ -0,0 +1,278 @@
+# OpenVINO™ Model Server Stateful Example Clients 
+
+This document contains examples to run *GetModelStatus*, *GetModelMetadata*, *Predict* functions over gRPC API and REST API.
+
+It covers following topics:
+* <a href="#grpc-api">gRPC API Stateful Client Example </a>
+* <a href="#rest-api">REST API Stateful Client Example </a>
+
+## Requirement
+
+Install client dependencies using the command below in the example_client directory:
+```
+pip3 install -r stateful_client_requirements.txt
+```
+
+### Getting ready with rm_lstm4f stateful model
+
+The purpose of this example is to show how to run stateful models on OpenVino model server.
+
+- Pre-requisite
+
+To run this example you will need to download the rm_lstm4f model with input and score ark files and convert it to IR format.
+ 1. Download the model from [rm_lstm4f](https://download.01.org/openvinotoolkit/2018_R3/models_contrib/GNA/rm_lstm4f/)
+
+ 2. Setup [OpenVINO](https://docs.openvinotoolkit.org/2019_R2/_docs_install_guides_installing_openvino_linux.html)
+
+ 3. Convert model to IR [How to convert](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_samples_speech_sample_README.html)
+
+ 4. Place ark files in the directories /home/download/rm_lstm4f/test_feat_1_10.ark /home/download/rm_lstm4f/test_score_1_10.ark
+
+Having rm_lstm4f model files .xml and .bin in the IR format present in /home/download/rm_lstm4f/model/1 directory,
+OVMS can be started using the command:
+```bash
+docker run -d -rm -v /home/rm_lstm4f/model:/tmp/model -p 8111:8111 -p 5555:5555 openvino/model_server:latest --stateful --port 8111 --rest_port 5555 --model_name rm_lstm4f --model_path /tmp/model
```

Comment:
in other example clients we have port 9000 as the default instead of 8111. It would be good to keep is consistent.